articles/openshift/oshift-how-add-domains-proxy-whitelist.md

```diff
@@ -0,0 +1,122 @@
+---
+title: How to permit outbound access to internet hosts in deployments with public services network connectivity | UKCloud Ltd
+description: Outlines methods to securely permit access to internet hosts in v3.11 OpenShift clusters deployed with public services network connectivity.
+services: openshift
+author: Ben Bacon
+toc_rootlink: How To
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Permit outbound internet access in deployments with public services network connectivity
+toc_fullpath: How To/oshift-how-add-domains-proxy-whitelist.md
+toc_mdlink: oshift-how-add-domains-proxy-whitelist.md
+---
+
+# How to permit outbound access to internet hosts in deployments with public services network connectivity
+
+## Overview
+
+This article outlines the necessary steps to whitelist domain names within Squid proxy. Squid is an enabled service within v3.11 OpenShift clusters deployed with connectivity to public sector networks (such as: HSCN, PSN-A, Janet). Whitelisted domains can be accessed on the Internet via the proxy enabling you to request external resources/data on nodes that previously only had connectivity to public sector networks.
+
+## Prerequisites
+
+To complete the steps in this guide, you must have access to a cluster which is running OpenShift Container Platform v3.11 (or newer) with connectivity to a public services network. Squid proxy is not currently enabled within environments that do not have access to a public services network.
+
+You must also have access to either:
+
+- the OpenShift Web console
```

Comment:
Add an empty line between bullet points otherwise it doesn't render properly in the generated HTML